docs/en/operations/access_rights.md

```diff
@@ -1,106 +1,65 @@
-# Access Rights {#access-rights}
+# Access Control and Account Management {#access-control}
 
-Users and access rights are set up in the user config. This is usually `users.xml`.
+ClickHouse support access control management based on [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) approach.
 
-Users are recorded in the `users` section. Here is a fragment of the `users.xml` file:
+Elements of ClickHouse access control model:
+- [User account](#user-account-management)
+- [Role](#role-management)
+- [Row Policy](#row-policy-management)
 
-``` xml
-<!-- Users and ACL. -->
-<users>
-    <!-- If the user name is not specified, the 'default' user is used. -->
-    <default>
-        <!-- Password could be specified in plaintext or in SHA256 (in hex format).
+!!! info "Note"
+    Some of the access control functionality is available through the [users.xml](settings/settings_users.md) server configuration files, but we don't recommend using this way of user permissions management. Don't combine these methods of access control management, or it may cause unexpected exceptions.
```

Comment:
Actually it's not very easy to combine them because SQL commands never change users.xml, even if you have all privileges.

The phrase "Unexpected exception" sounds like you've found a bug. Bugs should be fixed, not documented.